src/main/java/com/jcabi/github/mock/MkNotifications.java

```diff
@@ -40,24 +41,24 @@
  *
  * @author Giang Le (lthuangiang@gmail.com)
  * @author Paul Polishchuk (ppol@ua.fm)
+ * @author Piotr Pradzynski (prondzyn@gmail.com)
  * @version $Id$
  * @since 0.15
  * @see <a href="https://developer.github.com/v3/activity/notifications/">Notifications API</a>
- * @todo #920 Implement iterate() and get() operations in MkNotifications.
- *  Don't forget about unit tests.
  * @todo #913:30min Implement markAsRead() and thread() operations in
  *  MkNotifications. Don't forget about unit tests.
  */
 @Immutable
 final class MkNotifications implements Notifications {
+
     @Override
     public Iterable<Notification> iterate() {
-        throw new NotImplementedException("MkNotifications#iterate");
+        return Collections.emptyList();
```

Comment:
@prondzyn I think you should return a real collection instead empty one.
Let's create the collection inside the constructor, size of the collection can be a parameter.